src/d3d9/d3d9_adapter.cpp

```diff
@@ -531,7 +533,7 @@ namespace dxvk {
     // Max Vertex Blend Matrix Index
     pCaps->MaxVertexBlendMatrixIndex = 0;
     // Max Point Size
-    pCaps->MaxPointSize              = 256.0f;
+    pCaps->MaxPointSize              = limits.pointSizeRange[1];
```

Comment:
Are games going to be fine with this not being an integer value / power-of-two? Vulkan drivers will report the upper limit of their fixed-point representation here, which is generally something of the format `(2^n - 1)/(2^m)`.

Reply:
pCaps->MaxPointSize is a float, so you'd hope that games are fine if that's not an integer value. At least I don't have anything indicating they wouldn't be. We could ofc round it down to the nearest integer or power of two but right now I don't see any reason to do that so I'd prefer to wait until we hit a game that actually needs it.